sdk/healthinsights/azure-healthinsights-radiologyinsights/samples/sample_age_mismatch_inference.py

```diff
@@ -0,0 +1,154 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import datetime
+import os
+import uuid
+
+
+from azure.core.credentials import AzureKeyCredential
+from azure.healthinsights.radiologyinsights import RadiologyInsightsClient
+from azure.healthinsights.radiologyinsights import models
+
+"""
+FILE: sample_age_mismatch_inference.py
+
+DESCRIPTION:
+The sample_age_mismatch_inference.py module processes a sample radiology document with the Radiology Insights service.
+It will initialize a RadiologyInsightsClient, build a Radiology Insights request with the sample document,
+submit it to the client, RadiologyInsightsClient, build a Radiology Insights job request with the sample document,
+submit it to the client and display 
```

Comment:
> It will initialize a RadiologyInsightsClient, build a Radiology Insights request with the sample document,
submit it to the client, RadiologyInsightsClient, build a Radiology Insights job request with the sample document,
submit it to the client and display

Phrasing looks like it's repeating itself. Same issue (if one) in all samples.

Reply:
adjusted